lib/matplotlib/tests/test_figure.py

```diff
@@ -193,61 +192,27 @@ def test_gca():
         assert fig.add_axes() is None
 
     ax0 = fig.add_axes([0, 0, 1, 1])
```

Comment:
```suggestion
    # test that gca() picks up Axes created via add_axes()`
    ax0 = fig.add_axes([0, 0, 1, 1])
```

Reply:
Agreed on the 122/111 comment, bad squashing when I was removing the other pieces. Updates are there now.